llama.cpp

```diff
@@ -683,7 +685,7 @@ struct llama_model_loader {
         }
 
         if (use_mmap) {
-            mapping.reset(new llama_mmap(&file_loaders.at(0)->file));
+            mapping.reset(new llama_mmap(&file_loaders.at(0)->file, false));
```

Comment:
Note that the second parameter to `llama_mmap` here disables madvise/prefetching unconditionally.

Reply:
Yes, that is something that I want to fix for the final version but I'm not sure about the best approach.